src/NodeVisitor/RootNamespaceNormalizer.php

```diff
@@ -44,7 +44,7 @@ public function beforeTraverse(array $nodes)
             }
         }
         // Wrap all statements into the namespace block
-        $globalNamespaceNode = new Namespace_(new FullyQualified(''), array_slice($nodes, $lastDeclareOffset));
+        $globalNamespaceNode = new Namespace_(null, array_slice($nodes, $lastDeclareOffset));
```

Comment:
Is it ok to use `null` here instead of FQN? This may require some checks from outside...

Reply:
By the way: there is not big deal for clients 'case name of [ReflectionFileNamespace](https://github.com/goaop/parser-reflection/blob/master/src/ReflectionFileNamespace.php) is accessible only with [ReflectionFileNamespace::getName()](https://github.com/goaop/parser-reflection/blob/master/src/ReflectionFileNamespace.php#L245) method with such contents:

```php
    public function getName()
    {
        $nameNode = $this->namespaceNode->name;
        return $nameNode ? $nameNode->toString() : '';
    }
```

Reply:
Added a test case to validate behavior of `ReflectionFileNamespace::getName()` is still the same.